src/shared/wizards/multiStepWizard.ts

```diff
@@ -90,3 +95,54 @@ export class BrowseFolderQuickPickItem implements FolderQuickPickItem {
         return result[0]
     }
 }
+
+export async function promptUserForLocation(
+    context: WizardContext,
+    helpButton?: { button: vscode.QuickInputButton; url: string }
+): Promise<vscode.Uri | undefined> {
+    const items: FolderQuickPickItem[] = (context.workspaceFolders || [])
+        .map<FolderQuickPickItem>(f => new WorkspaceFolderQuickPickItem(f))
+        .concat([
+            new BrowseFolderQuickPickItem(
+                context,
+                localize(
+                    'AWS.samcli.initWizard.location.prompt',
+                    'The folder you select will be added to your VS Code workspace.'
```

Comment:
don't need to mention the editor name, just "your workspace" or "the workspace" is enough.